Tentacle/RepositoryInfo.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  RepositoryInfo.swift
+//  Tentacle
+//
+//  Created by Romain Pouclet on 2016-08-02.
+//  Copyright © 2016 Matt Diephouse. All rights reserved.
+//
+
+import Foundation
+import Argo
+import Curry
+
+public struct RepositoryInfo: Hashable, CustomStringConvertible {
+    /// The id of the repository
+    public let id: String
+    
+    /// The basic informations about the owner of the repository, either an User or an Organization
+    public let owner: Owner
```

Comment:
All the fields looked the same to me—even the field that indicates whether its an organization. So I think they're the same thing. We just need to add something to `User` that says whether its an organization.